CRM/Financial/BAO/PaymentProcessor.php

```diff
@@ -569,6 +569,30 @@ public static function getPaymentProcessorForRecurringContribution($contribution
     return Civi\Payment\System::singleton()->getById($paymentProcessorId);
   }
 
+  /**
+   * Get the name of the payment processor
+   *
+   * @param $paymentProcessorId
+   *
+   * @return null|string
+   */
+  public static function getPaymentProcessorName($paymentProcessorId) {
+    // if there is a payment processor ID, get the name of the payment processor
```

Comment:
I don't really think this ID check is necessary

Reply:
@omarabuhussein Ok, removed.